open-metadata-implementation/governance-servers/open-lineage-services/open-lineage-services-client/src/main/java/org/odpi/openmetadata/governanceservers/openlineage/client/OpenLineageClient.java

```diff
@@ -66,9 +68,10 @@ public LineageVerticesAndEdges lineage(String userId,
 
     private void detectExceptions(String methodName,
                                   LineageResponse response)
-            throws org.odpi.openmetadata.frameworks.connectors.ffdc.InvalidParameterException, PropertyServerException {
-        exceptionHandler.detectAndThrowInvalidParameterException(methodName, response);
-        exceptionHandler.detectAndThrowPropertyServerException(methodName, response);
+            throws org.odpi.openmetadata.frameworks.connectors.ffdc.InvalidParameterException, PropertyServerException, OpenLineageException {
```

Comment:
Please note that when backend server is up, but not configured as OLS, response is null and a NPE is thrown after detectExceptions method